src/Core/OntologyAnnotation.fs

```diff
@@ -42,8 +42,14 @@ type OntologyAnnotation(?name,?tsr,?tan, ?comments) =
 
     member this.TANInfo = 
         match this.TermAccessionNumber with
-        | Some v -> 
-            Regex.tryParseTermAnnotation v
+        | Some tan -> 
```

Comment:
this should be kind of reverse right?

We have one example the paff(?) subnamespace in the MS ontology. Using this as example we would have:

TSR: `MS`; TAN: `PEFF:0001032`

Then we want to return `{| IDSpace = MS; LocalID = PEFF:0001032 |}`

In my opinion we should give a explicitly set TSR priority.

Reply:
But then we would have `MS:PEFF:0001032` as the short-annotation.

Reply:
Ahh sorry my bad! Didnt fully understand the function context. All good from my side